components/fieldgroup/stories/fieldgroup.test.js

```diff
@@ -7,47 +6,51 @@ export const FieldGroupSet = Variants({
 	testData: [
 		{
 			testHeading: "Default",
+			inputType: "radio",
 		},
 		{
-			testHeading: "Horizontal",
+			testHeading: "Vertical checkboxes",
+			layout: "vertical",
+			inputType: "checkbox",
+			helpText: "Make a selection.",
+		},
+		{
+			testHeading: "Horizontal radios",
+			layout: "horizontal",
+			inputType: "radio",
+		},
+		{
+			testHeading: "Horizontal checkboxes",
 			layout: "horizontal",
-			items: [
-				(passthroughs, context) => Checkbox({
-					...passthroughs,
-					id: "apple",
-					label: "Apples are best",
-					customClasses: ["spectrum-FieldGroup-item"],
-				}, context),
-				(passthroughs, context) => Checkbox({
-					...passthroughs,
-					id: "banana",
-					label: "Bananas forever",
-					customClasses: ["spectrum-FieldGroup-item"],
-				}, context),
-				(passthroughs, context) => Checkbox({
-					...passthroughs,
-					id: "cherry",
-					label: "Cherries ftw",
-					customClasses: ["spectrum-FieldGroup-item"],
-				}, context),
-			],
-		},
-		{
-			testHeading: "Label position: side",
+			inputType: "checkbox",
+			helpText: "Make a selection.",
+		},
+		{
+			testHeading: "Radios label position: side",
```

Comment:
Let's also add the "Vertical radios: side label" and "Vertical checkboxes: side label" test cases too 👍 

<img width="400" alt="Screenshot 2024-10-17 at 12 40 25 PM" src="https://github.com/user-attachments/assets/818889dd-a8d7-42ec-92b4-33af2fa21022">